src/templates/users/user_detail.html

```diff
@@ -3,25 +3,43 @@
 {% load static %}
 
 {% block title %}
-  User:
-  {{ object.name }}
+  My Details:
 {% endblock title %}
 {% block content %}
   <div class="container">
     <div class="row">
       <div class="col-sm-12">
-        <h2>{{ object.name }}</h2>
+        <h2>My Details:</h2>
       </div>
     </div>
     {% if object == request.user %}
       <!-- Action buttons -->
+      <style>
+        .list-item {
+          margin-bottom: 20px;
+          list-style-type: none;
+        }
+      </style>
       <div class="row">
         <div class="col-sm-12">
-          <a class="btn btn-primary" href="{% url 'users:update' %}" role="button">My Info</a>
-          <a class="btn btn-primary"
-             href="{% url 'account_email' %}"
-             role="button">E-Mail</a>
-          <!-- Your Stuff: Custom user template urls -->
+          <ul>
+            <li class="list-item">
+              User Display Name: {{ object.name }}
+              <a class="btn btn-primary" href="{% url 'users:update' %}" role="button">Update Name</a>
+            </li>
+            <li class="list-item">
+              Email: {{ object.email }}
+              <a class="btn btn-primary"
+                 href="{% url 'account_email' %}"
+                 role="button">Update E-Mail</a>
+            </li>
+            <li class="list-item">Password: {{ object.password }}</li>
```

Comment:
it might be a good idea to not show the password and instead have a Change Password button that takes you to change password view that can be implemented in future issues